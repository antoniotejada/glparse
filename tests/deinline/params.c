void a(int param_int_0)
{

}

void f(int param_int_0)
{
    a(param_int_0);
    a(param_int_0);
    a(param_int_0);
    a(param_int_0);
    a(param_int_0);
    a(param_int_0);
}