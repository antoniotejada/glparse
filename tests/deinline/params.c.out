// Src lines 6
// Dest lines 5

void a(int param_int_0);
void f(int param_int_0);
void subframe2(int param_int_0);

void a(int param_int_0)
{
}

void f(int param_int_0)
{
    subframe2(param_int_0);
    subframe2(param_int_0);
    subframe2(param_int_0);
}

void subframe2(int param_int_0)
{
    a(param_int_0);
    a(param_int_0);
}
